bladeui/js/pages/baseClass.js

```diff
@@ -35,37 +36,69 @@ class BaseClass
       {
         this.handleChangeView(FIRST_PAGE);
       }
+      else
+      {
+        this.bearerToken = token;
+        request({
+          method: "get",
+          url: "/jwt/transfer/info",
+          headers: {
+            Authorization: `Bearer ${data.bladeUserData.token}`
+          }
+        })
+        .then(response =>
+        {
+          const res = JSON.parse(response.response);
+          this.transferPossibilityNotification = res.transfer_possibility === "ALLOWED";
+          if (this.transferPossibilityNotification)
+          {
+            this._renderTransferNotification();
+          }
+        })
+        .catch(err => console.error(err));
+      }
     });
   }
 
   initListeners()
   {
-
+    throw new Error("you mast implement this method");
```

Comment:
Please, specify why this error is thrown (in comment above). Or, if possible, use the TODO to not forget to implement this method

Reply:
The error is thrown because this method is required to implement in child classes